### DISCUSSION TITLE
Doesn't work on Poco F2 PRO.

### DISCUSSION THREAD

**Discussion by BahadirKaya90** [AppGround-io/bluetooth-keyboard-and-mouse-support#183] [Bugs & Problems]
I can't use the app on Poco F2 PRO. Does my phone need Bluetooth HID Profile for it to work? I checked my phone doesn't have Bluetooth HID Profile. Even though it seems it is connected there is no mouse cursor on the TV. I can USE Xiaomi MI 9T and MI 8 without problem and they have HID profile. Please reply.

**Comment by zorazorr** [comment]
Hello,
a HID Profile is necessary.
